### ISSUE TITLE
can only show 10000 variables

### ISSUE THREAD

**Issue by Firestar-Reimu** [lckr/jupyterlab-variableInspector#272]
I have a list of 15300 but [jupyterlab-variableInspector](https://github.com/lckr/jupyterlab-variableInspector) can only show 10000 variables

**Comment by lckr** [comment]
Yes, this is by design since we have to send each value we want to display in the variable inspector to the front end which can significantly slow down or even block the jupyter kernel each time you run a cell. You could change that behavior by adapting the code on the python side [here](https://github.com/lckr/jupyterlab-variableInspector/blob/master/src/inspectorscripts.ts#L191). The plan is to eventually expose this variable more easily on the frontend as well

**Comment by Firestar-Reimu** [comment]
Thanks, is it possible to set this variable in JupyterLab's Settings in the future?
